ceph_deploy/osd.py

```diff
@@ -706,46 +767,81 @@ def make_disk(parser):
     """
     Manage disks on a remote host.
     """
-    parser.add_argument(
-        'subcommand',
-        metavar='SUBCOMMAND',
-        choices=[
-            'list',
-            'prepare',
-            'activate',
-            'zap',
-            ],
-        help='list, prepare, activate, zap',
+    disk_parser = parser.add_subparsers(dest='subcommand')
+
+    disk_zap = disk_parser.add_parser(
+        'zap',
+        help='destroy existing partition table and content for DISK',
+        )
+    disk_zap.add_argument(
+        'disk',
+        nargs='+',
+        metavar='HOST:DISK',
+        type=colon_separated,
+        help='host and disk'
+        )
+
+    disk_list = disk_parser.add_parser(
+        'list',
+        help='List disk info from remote host(s)'
         )
-    parser.add_argument(
+    disk_list.add_argument(
         'disk',
         nargs='+',
         metavar='HOST:DISK',
         type=colon_separated,
-        help='host and disk (or path)',
+        help='remote host to list OSDs from'
         )
-    parser.add_argument(
+
+    disk_prepare = disk_parser.add_parser(
+        'prepare',
+        help='Prepare a disk for use as Ceph OSD by formatting/partitioning disk'
+        )
+    disk_prepare.add_argument(
         '--zap-disk',
         action='store_true', default=None,
         help='destroy existing partition table and content for DISK',
         )
-    parser.add_argument(
+    disk_prepare.add_argument(
         '--fs-type',
         metavar='FS_TYPE',
+        choices=['xfs',
+                 'ext4',
+                 'btrfs'
+                 ],
         default='xfs',
-        help='filesystem to use to format DISK (xfs, btrfs or ext4)'
+        help='filesystem to use to format DISK',
```

Comment:
why are we removing the file system types supported? I know we have that as a common question coming up: "can I use $FS here?" or "I tried using $FS and it didn't work"